jdaviz/configs/imviz/plugins/aper_phot_simple/aper_phot_simple.py

```diff
@@ -145,6 +155,7 @@ def vue_do_aper_phot(self, *args, **kwargs):
             npix = np.sum(aper_mask) * u.pix
             img = aper_mask.get_values(comp_no_bg, mask=None)
             aper_mask_stat = reg.to_mask(mode='center')
+            comp_no_bg_cutout = aper_mask_stat.cutout(comp_no_bg)

```

Comment:
@PatrickOgle et al. -- You do want radial profile of background subtracted data, right? If not, I can change this to plot raw data. Please confirm.